slack.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Slack
+permalink: /slack
+tags: about
+---
+
+> Want an invite? Anyone who donates to the [Soft Skills Engineering Patreon](https://www.patreon.com/SoftSkillsEng) gets an invite to the Slack.
```

Comment:
Can we mention that invites are sent out during the first week of each month after Patreon charges you?

Reply:
Done!